zavod/zavod/tests/test_exporters.py

```diff
@@ -0,0 +1,70 @@
+from zavod.exporters import export
+from zavod.context import Context
+from zavod.store import View, get_store, get_view
+from zavod.meta import Dataset
+from zavod.runner import run_dataset
+from zavod import settings
+from followthemoney.cli.util import read_entities
+from json import load, loads
+from csv import DictReader
+
+def test_export(vdataset: Dataset):
+    run_dataset(vdataset)
+    export(vdataset.name)
+
+    expected_resources = [
+        "entities.ftm.json",
+        "names.txt",
+        "senzing.json",
+        "source.csv",
+        "statistics.json",
+        "targets.nested.json",
+        "targets.simple.csv",
+    ]
+
+    dataset_path = settings.DATA_PATH / "datasets" / vdataset.name
+    with open(dataset_path / "entities.ftm.json") as ftm:
+        # it parses and finds expected number of entites
+        assert len(list(read_entities(ftm))) == 11
+
+    with open(dataset_path / "index.json") as index_file:
+        index = load(index_file)
+        assert index["name"] == vdataset.name
+        assert index["entity_count"] == 11
+        assert index["target_count"] == 8
+        resources = {r["name"] for r in index["resources"]}
+        for r in expected_resources:
+            assert r in resources
```

Comment:
how clean do you want to go with the exporter tests? would I be wasting time if I do a method per class, each running the dataset and doing detailed assertions there?

Or is it reasonable at this stage just to make functions that test these files in as much detail as we like in this one test method?

Reply:
I do think that having one overall test and one test that makes assertions on each format would be a good idea. This is the area of testing where we're most likely to catch customer-facing issues, so it's worth investing into.